From 13af48843dcff5c4ab9e7bc4622f7cb7ff0dd0a7 Mon Sep 17 00:00:00 2001
From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Mon, 16 Nov 2009 11:31:49 +0100
Subject: [PATCH] simple_none.c uses HAVE_SOFT_FLOAT it has to include config.h
 for this to work properly.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>

cherry-picked from alsa-lib.git master 13af48843dcff5c4ab9e7bc4622f7cb7ff0dd0a7

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>

---
 src/mixer/simple_none.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/mixer/simple_none.c b/src/mixer/simple_none.c
index 9259a08..426f2d7 100644
--- a/src/mixer/simple_none.c
+++ b/src/mixer/simple_none.c
@@ -40,6 +40,7 @@
 #include <limits.h>
 #include <alsa/asoundlib.h>
 #include "mixer_simple.h"
+#include "config.h"
 
 #ifndef DOC_HIDDEN
 
-- 
1.6.5.2

